src/main/java/org/lightcouch/View.java

```diff
@@ -583,7 +603,22 @@ public View tempView(MapReduce mapReduce) {
     }
 
     private String getKeyAsJson(Object... key) {
-        return (key.length == 1) ? gson.toJson(key[0]) : gson.toJson(key); // single or complex key
+        // single or complex key
+        if (key.length == 1) {
+            return gson.toJson(key[0]);
+        } else {
+            return gson.toJson(key);
+        }
+    }
+
+    private JsonElement getKeyAsJsonElement(Object... key) {
+        // single or complex key
+        // If the startKey is a single key value, grab the first value from the JSON tree array
+        if(key.length == 1) {
+            return gson.toJsonTree(key).getAsJsonArray().get(0);
```

Comment:
If there is only one object, is there an easier way to get the `JsonElement` besides moving through `JsonTree` and `JsonArray` before just getting the element?
i.e. doesn't  `gson.toJsonTree(key[0]);` work?